sources/packages/backend/libs/integrations/src/sfas-integration/sfas-integration.processing.service.ts

```diff
@@ -115,7 +115,10 @@ export class SFASIntegrationProcessingService {
               const errorDescription = `Error processing record line number ${record.lineNumber}`;
               this.logger.error(errorDescription);
               this.logger.error(error);
-              result.summary.push(errorDescription);
+              this.logger.error(error.detail);
```

Comment:
What is the difference between logging the error detail and error itself. Is `logger.error(error)` not sufficient?

Reply:
Same reason for https://github.com/bcgov/SIMS/pull/3633#discussion_r1720046437
The difference is we log the error detail to the queue-consumer terminal. I wanted to make sure the terminal has `error.detail`, but I forgot to add `error.message`. If we want to keep the original logging way, I can revert it back.

Reply:
On using result.summary.push, I see your intention to have the error details in the the summary. But this summary here is an obsolete object which will be eventually replaced by the new process summary framework. Ref: [New process summary](https://github.com/bcgov/SIMS/blob/main/sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/ecert-integration/ecert-part-time-feedback-integration.scheduler.ts#L35)